src/rules/no_unsafe_finally.rs

```diff
@@ -1,12 +1,38 @@
 // Copyright 2020-2021 the Deno authors. All rights reserved. MIT license.
-use super::{Context, LintRule, ProgramRef, DUMMY_NODE};
-use swc_common::Span;
-use swc_ecmascript::ast::Stmt::{Break, Continue, Return, Throw};
-use swc_ecmascript::ast::TryStmt;
-use swc_ecmascript::visit::{noop_visit_type, Node, VisitAll, VisitAllWith};
+use super::{Context, LintRule, ProgramRef};
+use crate::handler::{Handler, Traverse};
+use derive_more::Display;
+use dprint_swc_ecma_ast_view::{self as AstView, NodeTrait};
+use swc_common::{Span, Spanned};
 
 pub struct NoUnsafeFinally;
 
+const CODE: &str = "no-unsafe-finally";
+const HINT: &str = "Use of the control flow statements (`return`, `throw`, `break` and `continue`) in a `finally` block will most likely lead to undesired behavior. It's recommended to take a second look.";
```

Comment:
Please remove `It's recommended to take a second look.`. Also it seems that this line exceeds 120 chars, could it be split into multiline?

Reply:
Okay, fixed it!